packages/dds/tree/src/feature-libraries/chunked-forest/codec/chunkCodecUtilities.ts

```diff
@@ -176,8 +177,14 @@ export interface ChunkDecoder {
 	/**
 	 * Read from stream, updating the offset.
 	 *
+	 * For fields that support incremental encoding, the encoded data can be retrieved via `getIncrementalFieldBatch`.
+	 *
 	 * @returns a TreeChunk made from the data from `stream`.
 	 * This chunk is allowed to reference/take ownership of content it reads from the stream.
 	 */
-	decode(decoders: readonly ChunkDecoder[], stream: StreamCursor): TreeChunk;
+	decode(
+		decoders: readonly ChunkDecoder[],
+		stream: StreamCursor,
+		getIncrementalFieldBatch?: (fieldKey: string) => EncodedFieldBatch,
```

Comment:
If that's actually a field key, use the proper type to indicate that.

```suggestion
		getIncrementalFieldBatch?: (fieldKey: FieldKey) => EncodedFieldBatch,
```

Reply:
Having read further into this PR now, I believe this is not a field key, but rather summaryRefId. Either way, more clarity would be good.

Reply:
Updated it to be of type `ChunkReferenceId`